pkg/BUILD

```diff
@@ -8,6 +8,13 @@ exports_files(
     visibility = ["//visibility:public"],
 )
 
+constraint_setting(name = "not_compatible_setting")
+
+constraint_value(
+    name = "not_compatible",
+    constraint_setting = ":not_compatible_setting",
+)
+
```

Comment:
It looks like this could be replaced with `@platforms//:incompatible` in bazel 4.0 (https://github.com/bazelbuild/platforms/pull/20)

Reply:
Yes. But I want to be compatible with 3.7 for now